src/sentry/static/sentry/app/actionCreators/repositories.tsx

```diff
@@ -0,0 +1,32 @@
+import * as Sentry from '@sentry/browser';
+
+import RepoActions from 'app/actions/repositoryActions';
+import {Client} from 'app/api';
+import {Repository} from 'app/types';
+
+type ParamsGet = {
+  orgSlug: string;
+};
+
+export function getRepositories(api: Client, params: ParamsGet) {
+  const {orgSlug} = params;
+  const path = `/organizations/${orgSlug}/repos/`;
+
+  RepoActions.loadRepos(orgSlug);
+
+  return api
```

Comment:
Why not use `async/await` here?

Reply:
I'm following the common pattern from other `actionCreators`.